showcase/src/Model.elm

```diff
@@ -19,17 +21,27 @@ type alias Model =
     , checkboxesStories : Checkboxes.Model
     , radioStories : Radio.Model
     , tabsStories : Tabs.Model
+    , navigatorsStories : Navigators.Model 
     , tabs : TabsPlugin.Model
     }
 
 
 init : Model
 init =
+    let
+        renderConfig =
```

Comment:
It's better to receive this guy from Main.
(Would be even better if we were receiving from UIExplorer/subscriptions)